src/seedu/addressbook/Main.java

```diff
@@ -79,14 +79,20 @@ private void exit() {
     }
 
     /** Reads the user command and executes it, until the user issues the exit command.  */
-    private void runCommandLoopUntilExitCommand() {
+    private void runCommandLoopUntilExitCommand() throws RuntimeException {
```

Comment:
Would it be better to handle storage exceptions at the storage class itself? 
Recall from tutorial on ways to improve exception handling =).